rh/schema.py

```diff
@@ -6,6 +6,9 @@
 )
 from project.util.session_mutation import SessionFormMutation
 from project import schema_registry
+from project.common_data import Choices
+
+c = Choices.from_file('borough-choices.json')
```

Comment:
Oh, this is actually defined as [`BOROUGH_CHOICES` in `project/util/address_form_fields.py`](https://github.com/JustFixNYC/tenants2/blob/master/project/util/address_form_fields.py#L8), do you mind importing it from there instead?